library/src/main/java/io/github/luizgrp/sectionedrecyclerviewadapter/SectionedRecyclerViewAdapter.java

```diff
@@ -451,6 +451,58 @@ public int getPositionInAdapter(Section section, int position) {
     }
 
     /**
+     * Helper method that returns the position of header in the adapter.
+     *
+     * @param tag unique identifier of the section
+     * @return position of the header in the adapter. -1 if the section doesn't have header
+     */
+    public int getHeaderPositionInAdapter(String tag) {
+        Section section = getValidSectionOrThrowException(tag);
+
+        return getHeaderPositionInAdapter(section);
+    }
+
+    /**
+     * Helper method that returns the position of header in the adapter.
+     *
+     * @param section a visible section of this adapter
+     * @return position of the header in the adapter. -1 if the section doesn't have header
+     */
+    public int getHeaderPositionInAdapter(Section section) {
+        if (!section.hasHeader) {
+            return -1;
+        }
+
+        return getSectionPosition(section);
+    }
+
+    /**
+     * Helper method that returns the position of footer in the adapter.
+     *
+     * @param tag unique identifier of the section
+     * @return position of the footer in the adapter. -1 if the section doesn't have footer
+     */
+    public int getFooterPositionInAdapter(String tag) {
+        Section section = getValidSectionOrThrowException(tag);
+
+        return getFooterPositionInAdapter(section);
+    }
+
+    /**
+     * Helper method that returns the position of header in the adapter.
+     *
+     * @param section a visible section of this adapter
+     * @return position of the footer in the adapter. -1 if the section doesn't have footer
+     */
+    public int getFooterPositionInAdapter(Section section) {
+        if (!section.hasFooter) {
+            return -1;
```

Comment:
Please throw an IllegalStateException with message "Section doesn't have a footer" and update javadoc accordingly.
Please also add a unit test to check if this method throws the exception.